File: section75/src/main/java/org/packt/springboot22/vid07/model/Harvest.java
package org.packt.springboot22.vid07.model;

import java.time.LocalDate;

import org.springframework.data.annotation.Id;
import org.springframework.data.relational.core.mapping.Column;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Harvest {

	@Id
	//@GeneratedValue(strategy = GenerationType.SEQUENCE)
	private Long id;
	private String name;
	private Integer qty;
	private Float price;
	private String description;
	private String status;
	private String type;
	@Column(value="approved")
	private LocalDate approvedDate;
	
	

}

File: section67/src/main/kotlin/org/packt/springboot22/vid06/model/Harvest.kt
package org.packt.springboot22.vid06.model

import org.springframework.data.annotation.Id
import org.springframework.data.relational.core.mapping.Table
import java.time.LocalDate
import org.springframework.data.relational.core.mapping.Column

@Table("harvest")
data class Harvest (
	
	@Id val id : Long? = null,
	var name : String,
	var qty : Int,
	var price : Float,
	var description : String,
	var status : String,
	var type : String,
	@Column("approved") var approvedDate : LocalDate
)